app/views/upload.scala.html

```diff
@@ -29,15 +37,19 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
             <div class="govuk-form-group">
                 <div class="drag-and-drop">
                     <div class="govuk-summary-list">
-                        <div class="govuk-summary-list__row hide">
-                            <dd class="govuk-summary-list__value">
+                        <div class="govuk-summary-list__row">
+                            <dd class="govuk-summary-list__value drag-and-drop__success hide">
                                 @greenTickMark()
                                 The folder
                                 "<span id="folder-name"></span>"
                                 (containing
                                 <span id="folder-size"></span>)
                                 has been selected
                             </dd>
+                            <dd class="govuk-summary-list__value drag-and-drop__failure hide">
```

Comment:
When I test with an empty folder I get the error message: "You can only drop a single folder". Is that correct? I would have thought we need a different error message depending on the scenario, ie:

- User tries to drag and drop a non-folder: "You can only drop a single folder"
- User tries to drag and drop empty folder: "You cannot drop an empty folder"

Has the wording of the error message(s) been confirmed with Sarri, Rooha and Kirsten?

Reply:
This is something I brought up too. I think this will be decided by Sarra, Rooha and Kirsten after the user testing.

Reply:
Yes, thanks for doing that.